src/ResourceManager/RedisCache/Commands.RedisCache/Commands.RedisCache.csproj

```diff
@@ -161,4 +205,9 @@
     </ItemGroup>
     <Copy SourceFiles="@(MarkdownFiles)" DestinationFolder="$(OutputPath)\help\" ContinueOnError="false" />
   </Target>
+  <Import Project="..\..\..\packages\Microsoft.Bcl.Build.1.0.14\tools\Microsoft.Bcl.Build.targets" Condition="Exists('..\..\..\packages\Microsoft.Bcl.Build.1.0.14\tools\Microsoft.Bcl.Build.targets')" />
```

Comment:
Please delete these 5 lines to fix the build.

Reply:
I have removed the Bcl imports to fix the build. I am not sure though why it failed because of Bcl package.